src/Microsoft.Extensions.SecretManager.Tools/Internal/ProjectIdResolver.cs

```diff
@@ -100,7 +100,6 @@ private void EnsureProjectExtensionTargetsExist(string projectFile)
             using (var stream = new FileStream(projectExtensionsPath, FileMode.Create))
             using (var writer = new StreamWriter(stream))
             {
-                writer.WriteLine("<!-- Auto-generated by dotnet-user-secrets. This file can be deleted and should not be commited to source control. -->");
```

Comment:
Despite what you think, this was added to the end of the file. It doesn't add much value since the file is in 'obj/' directory, which is almost always excluded, repeatedly deleted, etc.